app/views/tournaments/show.html.erb

```diff
@@ -71,11 +78,11 @@
     </div>
 <% end %>
 
-<% if @tournament.tournament_type == "round robin" %>
+<% if @tournament.round_robin? %>
     <%= render "round_robin_tournament" %>
-<% elsif @tournament.tournament_type == "single elimination" %>
+<% elsif @tournament.single_elimination? %>
     <%= render "single_elimination_tournament" %>
-<% elsif @tournament.tournament_type == "multiplayer game" %>
+<% elsif @tournament.multiplayer_game? %>
     <%= render "multiplayer_game_tournament" %>
 <% else %>
     <%= render "king_of_the_hill_tournament" %>
```

Comment:
How about eliminating this whole sequence of `if`s and replace it with a simple
```
<%= render "#{@tournament.tournament_type}_tournament" %>
```
and just be sure through validation that the variable can only have values with matching partials?